File: Python/6502.py
r, w, l = map(int, input().split())
tc = 1
while r != 0:
	if (2 * r)**2 >= w**2 + l**2:
		print("Pizza %d fits on the table." % (tc))
	else:
		print("Pizza %d does not fit on the table." % (tc))
	tc += 1
	r, w, l = map(int, input().split())

	

File: C/6502.c
#include <stdio.h>
#include <math.h>

int main()
{
	int r, w, l, tc = 1;
	scanf("%d %d %d", &r, &w, &l);
	while (r != 0){
		if (pow(2*r, 2) >= pow(w, 2) + pow(l, 2)){
			printf("Pizza %d fits on the table.\n", tc);
		}
		else
			printf("Pizza %d does not fit on the table.\n", tc);
		tc += 1;
		scanf("%d %d %d", &r, &w, &l);
	}
	return 0;
}